package.yaml

```diff
@@ -1,48 +1,41 @@
 name:                servant-elm-street
 version:             0.1.0.0
-github:              "githubuser/servant-elm-street"
+github:              "ChaosGroup//servant-elm-street"
 license:             BSD3
-author:              "Author name here"
-maintainer:          "example@example.com"
-copyright:           "2021 Author name here"
+author:              "Magdalena Lungolova"
+maintainer:          "mvlungolova@gmail.com"
+copyright:           "Copyright 2021 Chaos Software OOD"
 
 extra-source-files:
 - README.md
-- ChangeLog.md
 
-# Metadata used when publishing your package
-# synopsis:            Short description of your package
-# category:            Web
-
-# To avoid duplicated efforts in documentation and dealing with the
-# complications of embedding Haddock markup inside cabal files, it is
-# common to point users to the README.md file.
-description:         Please see the README on GitHub at <https://github.com/githubuser/servant-elm-street#readme>
-
-dependencies:
-- base >= 4.7 && < 5
+description:         Please see the README on GitHub at <https://github.com/ChaosGroup/servant-elm-street/blob/main/README.md>
 
 library:
   source-dirs: src
 
 dependencies:
 - servant-foreign
 - text
-- servant-elm
 - base >= 4.7 && < 5
 - aeson
-- casing
 - elm-street
-- filepath
 - servant
-- servant-server
 - wai
-- warp
-- servant-js
 - random
```

Comment:
What do you need `random` f or?

Reply:
it was for an experiment long ago, I removed it now

Reply:
done